### PR TITLE
sentinel-tick, a tool to fetch fil prices and put them in sentinel

### PR DESCRIPTION
**Pull request by hsanjuan**
Initial commit.

### CODE REVIEW

File: model.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+// Quote stores FIL price information.
+type Quote struct {
+	//lint:ignore U1000 hit for go-pg
+	tableName struct{} `pg:"filquotes.fil_usd_quotes"`
+	Height    int64    `pg:",pk,use_zero,notnull"`
+	Price     float64  `pg:",use_zero,notnull"`
+	Exchange  string   `pg:",pk,use_zero,notnull`
+	Currency  string   `pg:",pk,use_zero,notnull`
+}
```
Comment (hsanjuan) [inline model.go:20]: @iand would this be ok for schema?

Comment (iand) [reply model.go:20]: I wouldn't use a float for currency. Use an int64 to represent the smallest unit you care about in the currency, e.g. milli cents

File: model.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	"github.com/filecoin-project/sentinel-tick/quotetracker"
+	pg "github.com/go-pg/pg/v10"
+)
+
+const schema = "filquotes"
+
+// Quote stores FIL price information.
+type Quote struct {
+	//lint:ignore U1000 hit for go-pg
+	tableName struct{} `pg:"filquotes.fil_usd_quotes"`
```
Comment (iand) [inline model.go:15]: I think this was where @placer14 was suggesting using a version suffix, e.g. `fil_usd_quotes_v1`

Comment (iand) [reply model.go:15]: Also table is called fil_usd_quotes but has a currency column?

**Review by iand — APPROVED** [review]

**Pull request merged**